lib/client/overlays/progress.js

```diff
@@ -121,6 +126,17 @@ const init = (options, socket) => {
       addCss(css);
       addHtml(html);
     });
+
+    document.addEventListener('visibilitychange', () => {
+      if (!document.hidden && hideOnPageVisible) {
+        const svg = document.querySelector(`#${ns}`);
+
+        // We need this timeout when page becomes visible
+        // otherwise we'll see a flash
+        resetTimer = setTimeout(() => reset(svg), 250);
```

Comment:
is `250` the optimum amount of time to wait for this scenario? (I'd like to comment that in code if so)

Reply:
Just an arbitrary delay that is long enough to let the user see HMR update is done when page becomes visible. Suggestions welcome.